### DISCUSSION TITLE
TailwindCSS getting in the way of content-collections

### DISCUSSION THREAD

**Discussion by RickyRaveanu** [sdorra/content-collections#252] [Q&A] [answered]
Hi everyone! I'm new to using content-collections and I'm really enjoying it so far. I've encountered an issue when trying to transform MDX content in my project. I'm using Tailwind CSS, and it seems that Tailwind's default settings are stripping away the styles from standard HTML elements, leaving only the JSX part of the MDX content styled.

Has anyone found a way to fix this issue without needing to manually add styles or create a custom field for every piece of content? Any help would be greatly appreciated!

Thank you!
For reference that's the result without & with TailwindCSS
<img width="678" alt="Screenshot 2024-08-18 at 4 02 25 AM" src="https://github.com/user-attachments/assets/8c2ea4a3-3ef7-46be-be80-deff6298ad89">
<img width="787" alt="Screenshot 2024-08-18 at 4 03 16 AM" src="https://github.com/user-attachments/assets/8950d189-cc03-49dd-b7da-f838a0c9ae81">

**Comment by sdorra** [comment]
@RickyRaveanu can you post the content of the mdx file?

**Reply by RickyRaveanu** [reply]
this is the same MDX content I had when I tried to use content-layer, and then found out it's not being maintained anymore :(
```
---
title: "Welcome to My Demo MDX Page in English"
summary: English Testing
---

import { useState } from 'react';

# Welcome to My Demo MDX Page

This is a simple demo of an MDX document.

## What is MDX?

MDX is a markup format that lets you seamlessly use JSX in your Markdown documents. You can import components, like React components, and use them in your Markdown.

## Interactive Counter Example

Below is a simple interactive counter implemented in React.

```jsx
const Counter = () => {
  const [count, setCount] = useState(0);

  return (
    <div>
      <h2>{count}</h2>
      <button onClick={() => setCount(count + 1)}>Increase</button>
      <button onClick={() => setCount(count - 1)}>Decrease</button>
    </div>
  );
};

<Counter />
```
```

**Reply by sdorra** [reply, marked as answer]
The mdx you showed does not contain a single tailwind class. There is nothing which could be stripped out by content-collections. I suspect that you used [tailwind typography plugin](https://github.com/tailwindlabs/tailwindcss-typography) and the class `prose` got lost during the migration. The `prose` class should wrapped around the mdx content e.g.:

```tsx
<article className="prose">
  <h1>{post.title}</h1>
  <MDXContent code={post.mdx} />
</article>
```

**Reply by RickyRaveanu** [reply]
Thank you for replying. I think my problem here is just tailwind itself. Because when I remove it, the MDX gets rendered how it should.

**Reply by RickyRaveanu** [reply]
the prose class actually solved it
```html
<article className="mx-auto max-w-4xl py-8 prose">
    <div className="mb-8 text-center">
      <h1 className="text-3xl font-bold">{ressource.title}</h1>
      <BreadcrumbComponent/>
    </div>
    <div dangerouslySetInnerHTML={{__html: ressource.body}}/>
  </article>
```
